### ISSUE TITLE
[one-cmds] Local test failure

### ISSUE THREAD

**Issue by seanshpark** [Samsung/ONE#10979]
With master branch b145f5c15031caca769fda4d7b21dbf8f0817d2c
```
one-quantize_016.test FAILED
one-quantize_neg_021.test FAILED
onecc_045.test FAILED
onecc_neg_026.test FAILED
```

**Comment by seanshpark** [comment]
`one-quantize_016.log`
```
circle_quantizer: [assert failed] Only support int32 CircleConst. 
circle_quantizer: [assert failed] Only support int32 CircleConst. 
circle_quantizer: ERROR: Internal Exception. BatchMatMul has wrong shape [.../compiler/luci/service/src/CircleShapeInferenceRule.cpp:476]
```
`one-quantize_neg_021.log` : same error log
`onecc_045.log` : same error log
`onecc_neg_026.log` : same error log

**Comment by seanshpark** [comment]
CC @jinevening

**Comment by jinevening** [comment]
Hmm.. The failed tests use `reshape_matmul.circle`, which is generated by `one-import onnx`.

**Comment by jinevening** [comment]
I couldn't reproduce the error. @seanshpark Could you share `reshape_matmul.circle` generated in your test directory?

**Comment by seanshpark** [comment]
@jinevening , thanks for verification!
I've tested twice with clean-reinstalling venv and test folders and got this result.
Now I've clean-reinstalled venv and test folders again and got all SUCCESS ... -_-;

Something fishy is that this error happened with internal onnx to circle tool with `one-import-onnx-ext` but I checked it didn't exist in bin folder when I began this test. I don't understand why this ghost like problem happened...
